suite-native/module-authorize-device/src/screens/passphrase/PassphraseFeatureUnlockFormScreen.tsx

```diff
@@ -37,6 +37,7 @@ export const PassphraseFeatureUnlockFormScreen = () => {
                 </Text>
                 <PassphraseForm
                     inputLabel={translate('modulePassphrase.form.createWalletInputLabel')}
+                    noPassphraseEnabled
```

Comment:
So now it will always show that "No Passphrase" button? I am not sure if that is something we want and could have some implications to desing/product. Also why is it necessary?

cc @matejkriz

Reply:
It's necessary since when this screen pops up based on the passphrase event, it's not possible to proceed with an empty passphrase

Reply:
Sure we could do that. I was thinking of adding the state to Redux, but that would require more changes...

Reply:
If we already have this information in navigation state there is no value adding it into redux

Reply:
The issue is that when called from these places it's in the context of AuthorizeDeviceStack, any ideas?

Reply:
Ok nevermind, I solved it by using `navigation.getParent()`